src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -615,7 +626,7 @@ public void SetTopmost(bool value)
         protected virtual IntPtr CreateWindowOverride(ushort atom)
         {
             return CreateWindowEx(
-                0,
+                (int)WindowStyles.WS_EX_NOREDIRECTIONBITMAP, // TODO this is only when using Win UI Comp.
```

Comment:
TODO... only add this style if... windows10 with win ui comp... and dx11 interface.